CI/dartboard-deploy.groovy

```diff
@@ -0,0 +1,346 @@
+#!groovy
+// Declarative Pipeline Syntax
+@Library('qa-jenkins-library') _
+
+def agentLabel = 'jenkins-qa-jenkins-agent'
+if (params.HARVESTER_KUBECONFIG) {
+    agentLabel = 'vsphere-vpn-1'
+}
+
+def scmWorkspace
+def generatedNames
+def configDirPath
+def parseToHTML(text) { text.replace('&', '&amp;').replace('<', '&lt;').replace('>', '&gt;') }
+
+def runningContainerName
+
+pipeline {
+  agent { label agentLabel }
+
+  environment {
+    // Define environment variables here.  These are available throughout the pipeline.
+    imageName = 'dartboard'
+    qaseToken = credentials('QASE_AUTOMATION_TOKEN')
+    qaseEnvFile = '.qase.env'
+    harvesterKubeconfig = 'harvester.kubeconfig'
+    templateDartFile = 'template-dart.yaml'
+    renderedDartFile = 'rendered-dart.yaml'
+    envFile = ".env"
+    DEFAULT_PROJECT_NAME = "${JOB_NAME.split('/').last()}-${BUILD_NUMBER}"
+    accessDetailsLog = 'access-details.log'
+    summaryHtmlFile = 'summary.html'
+    s3BucketName='jenkins-bullseye-storage'
+    s3BucketRegion='us-east-2'
```

Comment:
Not a blocker, but would be nice if these were parameterized, maybe with a default of these values.